sdl_android/src/main/res/values/sdl.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="sdl_router_service_version_name">sdl_router_version</string>
-    <integer name="sdl_router_service_version_value">5</integer>
+    <string name="sdl_router_service_version_name" translatable="false">sdl_router_version</string>
+    <integer name="sdl_router_service_version_value" translatable="false">5</integer>
```

Comment:
I don't think "translatable" applies to integers.

Reply:
Hi @MongoTheGeek Are you able to make the changes?

Reply:
Sorry. I have been back packing across Spain. I can make the changes in a couple of days.